core/finality-grandpa/primitives/src/lib.rs

```diff
@@ -172,6 +179,6 @@ decl_runtime_apis! {
 		/// When called at block B, it will return the set of authorities that should be
 		/// used to finalize descendants of this block (B+1, B+2, ...). The block B itself
 		/// is finalized by the authorities from block B-1.
-		fn grandpa_authorities() -> Vec<(AuthorityId, AuthorityWeight)>;
+		fn grandpa_authorities() -> AuthorityList;
```

Comment:
Could this method be removed? There's dedicated well-known storage key, so why keep it?

Reply:
Yes, I suspect so. Good point!